apps/squzy_storage/config/config.go

```diff
@@ -108,15 +116,59 @@ func New() Config {
 			withDbLog = value
 		}
 	}
+
+	dbType := DB_TYPE_CLICKHOUSE
```

Comment:
os.getenv has default option

os.Getenv(ENV_DB_LOGS, DB_TYPE_CLICKHOUSE)

Reply:
Hm, in default os package it is `func Getenv(key string) string`

Reply:
@DreamAndDrum ahh, anyways it can be a small local function